.github/pull_request_template.md

```diff
@@ -8,6 +8,7 @@
 - `yarn test:unit:all` should run without errors or warnings
 - `yarn serve` should run without errors or warnings
 - `yarn build` should run without errors or warnings
```

Comment:
* First/last name search does not work for me. Intended/expected?

Reply:
I had you set up as an Occupational Therapy user but the UI feature to support multiple compacts is actually still in the PR you're reviewing from John. I just went in and added you to the Audiology/Speech Language Pathology compact as well so you can look at those records in the meantime. Does that resolve the issue for you?

Reply:
Hmm, I'm actually testing on my own deployment on which I'm set up as an ASLP user. Same issue on the test site, too, though.

See - https://scribehow.com/shared/Search_for_Licensing_Data_by_Last_Name__T5LwE1-qTBOkN3cj7TRL1w

Reply:
Ok, I see that behavior in test as well. I'll have to do more digging. This is definitely not expected behavior and, most perplexing, I have test cases covering this and they do not behave this way.

Reply:
Found the issue. Short version, the code is fine but the feature update didn't get deployed to the CSG `test` environment yet because some merge wonkiness (resulting a missing test dependency in a lambda) caused the deploy to fail. I'll open a bugfix PR to get that dependency added back in.

Reply:
(See #241 )